tests/test_spdx_matcher.py

```diff
@@ -151,19 +173,21 @@ def test_fuzzy_match_matchConfidence_lt_1(self):
         """Test fuzzy match for license data attribute matchConfidence less than 1 in cache data."""
         _, cache_data = spdx_matcher._load_license_analyser_cache()
 
-        need = 5
-        for license_id, data in cache_data["licenses"].items():
-            if need == 0:
-                break
-            if data["matchConfidence"] < 1:
-                # exact match without any licenses, but fuzzy match get some of the licenses
-                analysis, _ = spdx_matcher.analyse_license_text(data["licenseText"])
-                self.assertTrue(len(analysis["licenses"]) == 0)
-
-                fuzzy_result = spdx_matcher.fuzzy_license_text(data["licenseText"], threshold=0.95)
-                self.assertTrue(len(fuzzy_result) > 0)
-                self.assertTrue(license_id in [match_license["id"] for match_license in fuzzy_result])
-                need -= 1
+        for match_config in (HeaderMatcher(), TextMatcher()):
```

Comment:
Add header matcher for the case `analyse_license_text` have no match result but `fuzzy_license_text` have